From 8347252a6c255f85f26d77f1e0d48336c949e081 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Mon, 23 Jan 2023 09:47:25 +0100
Subject: [PATCH] meson: install rasign2.1 man page too ##build

This file is installed by the GNU autotools-based build system but
not by meson.
---
 meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meson.build b/meson.build
index eef6b864d7..74025e2612 100644
--- a/meson.build
+++ b/meson.build
@@ -679,6 +679,7 @@ if cli_enabled
     'man/ragg2.1',
     'man/rahash2.1',
     'man/rarun2.1',
+    'man/rasign2.1',
     'man/rasm2.1',
     'man/rax2.1',
     'man/ravc2.1',
